src/pointer.cr

```diff
@@ -510,7 +510,7 @@ struct Pointer(T)
   # ditto
   def to_slice_null_terminated
     count = 0
-    until self[count].null?
+    while self[count]
```

Comment:
I honestly prefer the old version. It reads far better.

Reply:
I like the `while !self[count].null?` but isn't this 'de stijl'?